index.js

```diff
@@ -33,7 +33,7 @@ module.exports = function nodeExternals(options) {
     }
 
     // create the node modules list
-    var nodeModules = modulesFromFile ? utils.readFromPackageJson(options.modulesFromFile) : utils.readDir(modulesDir).filter(isNotBinary);
+    var nodeModules = modulesFromFile ? utils.readFromPackageJson(modulesFromFile) : utils.readDir(modulesDir).filter(isNotBinary);
```

Comment:
`readFromPackageJson` expects an object, with this change it will always receive a boolean